src/app/fdctl/run/tiles/fd_verify.h

```diff
@@ -40,6 +40,8 @@ typedef struct {
   ulong       out_chunk;
 } fd_verify_ctx_t;
 
+#define DEDUP_TAG_FROM_PAYLOAD_SIG(payload_sig_p) FD_LOAD( ulong, (payload_sig_p) )
```

Comment:
A macro defined in a header should be prefixed, though it's fine if it's just defined in the .c file. `FD_VERIFY_...`